src/content/8/en/part8c.md

```diff
@@ -57,7 +57,7 @@ module.exports = mongoose.model('Person', schema)
 ```
 
 
-We also included a few validations. _required: true_, which ensures that value exists, which is actually redundant by using GraphQL we ensure that the fields exist. However it is good to also keep validation in the database. 
+We also included a few validations. _required: true_, which makes sures that a value exists, is actually redundant: we already ensure that the fields exist with GraphQL. However, it is good to also keep validation in the database. 
```

Comment:
Which make sure

Reply:
Well-spotted! Fixed it; sorry about that. :)